cobc/cobc.c

```diff
@@ -3720,16 +3745,20 @@ process_command_line (const int argc, char **argv)
 			}
 			break;
 
-		case 6:
-			/* -fdefaultbyte=<xx> : Default initialization byte */
-			n = cobc_deciph_optarg (cob_optarg, 1);
-			if (n < 0 || n > 255) {
-				cobc_err_exit (COBC_INV_PAR, "-fdefaultbyte");
+		case CB_FLAG_GETOPT_TTITLE: /* 6 */
+			/* -fttitle=<title> : Title for listing */
+			{
+				size_t i, len;
+				cb_listing_with_title = cobc_main_strdup (cob_optarg);
```

Comment:
misses the check if it is already set, then free that we have for `-t` as the memory is not "leaked" we may drop this later but should be consistent within the same function about that

Reply:
done